docs/deploying/rhoai.md

```diff
@@ -0,0 +1,105 @@
+        metadata:
+          name: vllm-spyre-runtime
+          annotations:
+            openshift.io/display-name: vLLM IBM Spyre ServingRuntime for KServe
+            opendatahub.io/recommended-accelerators: '["ibm.com/aiu_pf"]'
+          labels:
+            opendatahub.io/dashboard: "true"
+        spec:
+          multiModel: false
+          supportedModelFormats:
+            - autoSelect: true
+              name: vLLM
+          containers:
+            - name: kserve-container
+              image: quay.io/ibm-aiu/vllm-spyre:latest.amd64
+              args:
+                - /mnt/models
+                - --served-model-name={{.Name}}
+              env:
+                - name: HF_HOME
+                  value: /tmp/hf_home
+                # Static batching configurations can also be set on each InferenceService
+                - name: VLLM_SPYRE_WARMUP_BATCH_SIZES
+                  value: '4'
+                - name: VLLM_SPYRE_WARMUP_PROMPT_LENS
+                  value: '1024'
+                - name: VLLM_SPYRE_WARMUP_NEW_TOKENS
+                  value: '256'
+              ports:
+                - containerPort: 8000
+                  protocol: TCP
+      ```
+
+2. Create an InferenceService for each model you want to deploy. This example demonstrates how to deploy `ibm-granite/granite-3.1-8b-instruct`.
+
+      ```yaml
+      apiVersion: serving.kserve.io/v1beta1
+      kind: InferenceService
+      metadata:
+        annotations:
+          openshift.io/display-name: granite-3-1-8b-instruct
+          serving.kserve.io/deploymentMode: RawDeployment
+        name: granite-3-1-8b-instruct
+        labels:
+          opendatahub.io/dashboard: 'true'
+      spec:
+        predictor:
+          imagePullSecrets:
+            - name: oci-registry
+          maxReplicas: 1
+          minReplicas: 1
+          model:
+            modelFormat:
+              name: vLLM
+            name: ''
+            resources:
+              limits:
+                ibm.com/aiu_pf: '1'
+              requests:
+                ibm.com/aiu_pf: '1'
+            runtime: vllm-spyre-runtime
+            storageUri: 'oci://registry.redhat.io/rhelai1/modelcar-granite-3-1-8b-instruct:1.5'
+            volumeMounts:
+              - mountPath: /dev/shm
+                name: shm
+          schedulerName: aiu-scheduler
+          tolerations:
+            - effect: NoSchedule
+              key: ibm.com/aiu_pf
+              operator: Exists
+                    spec:
+          volumes:
+            # This volume may need to be larger for bigger models and running tensor-parallel inference with more cards
+            - name: shm
+              emptyDir:
+                medium: Memory
+                sizeLimit: "2Gi"
+      ```
+
+3. Deploy and Test
+
+      Apply the manifests using `oc apply -f <filename>`:
+
+      ```console
+      oc apply -f servingruntime.yaml
+      oc apply -f inferenceservice.yaml
+      ```
```

Comment:
I think you could just include these `apply` lines in each step above after defining the manifests and then use this step for a  "Perform an inference request" example.

Reply:
👍 
I went ahead with the heredoc pattern in each section above (`oc apply -f - <<EOF`) and just linked out to the relevant kserve docs here for how to setup inference with a vllm deployment so we don't have to repeat that info